src/calculator.js

```diff
@@ -1,47 +1,29 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
+exports._check = (x,y) => {
+    if (typeof x !== 'number') {
+    throw new TypeError(`${x} is not a number`);
```

Comment:
Expected indentation of 6 spaces but found 4.

###### See http://eslint.org/docs/rules/indent for details about this rule